### DISCUSSION TITLE
Advisable to update to v89?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#992] [Q&A] [answered]
Google's floc is enabled in v89 and I don't see it being disabled in bromite in changeling. Should I update or wait for csagan to fix it?

**Comment by wchen342** [comment, marked as answer]
Isn't is disabled by default? [https://chromium.googlesource.com/chromium/src/+/refs/tags/89.0.4389.86/chrome/common/chrome_features.cc#720](https://chromium.googlesource.com/chromium/src/+/refs/tags/89.0.4389.86/chrome/common/chrome_features.cc#720)
